sdks/java/io/amazon-web-services2/src/main/java/org/apache/beam/sdk/io/aws2/sns/SnsIO.java

```diff
@@ -281,22 +281,33 @@ public Write<T> withRetryConfiguration(RetryConfiguration retry) {
     }
 
     /**
-     * Encode the full {@code PublishResult} object, including sdkResponseMetadata and
```

Comment:
You might use `@link` here instead of `@code`. That way it would have raised an error for the bad reference